hack/os-packages/Makefile

```diff
@@ -23,7 +23,7 @@ download-rpms:
 	$(MAKE) rhel_major_version=$(rhel_major_version) fips=$(fips) rpm-artifacts-getter.check
 	$(MAKE) rhel_major_version=$(rhel_major_version) fips=$(fips) packages="audit ca-certificates conntrack-tools chrony curl ebtables open-vm-tools python3-pip python-netifaces python-requests socat sysstat yum-utils yum-plugin-versionlock nfs-utils NetworkManager" download-rpm
 	# packages that are not in CentOS-7 Minimal
-	$(MAKE) rhel_major_version=$(rhel_major_version) fips=$(fips) packages="tzdata nspr nss python-chardet yum-utils nss-softokn-freebl nss-util nss-softokn nss nss-tools libxml2-python python-kitchen yum-plugin-ovl" download-rpm
+	$(MAKE) rhel_major_version=$(rhel_major_version) fips=$(fips) packages="tzdata nspr nss python-chardet yum-utils nss-softokn-freebl nss-util nss-softokn nss nss-tools  NetworkManager-libnm libdrm libpciaccess libxml2-python python-kitchen yum-plugin-ovl" download-rpm
```

Comment:
What happens if you remove this whole line do these packages get downloaded because they are dependencies of other packages that are being downloaded? We do not have a clear way of generating this list of images, it would be ideal if this Makefile only needed to specify the packages that are explicitly installed in the Ansible code.

Reply:
I tested it in offline mode so the missing dependency is not downloaded and I had to add those. We can do `yum deplist <required rpms>` and compile list of rpm required by comparing it with rpms already available in minimum iso + `core` packages. I can give it a try.